Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/FastDepthAwareBlurCommon.azsli

```diff
@@ -109,7 +109,7 @@ ShaderResourceGroup PassSrg : SRG_PerPass
         // The higher this value, the sharper edges will appear
         float m_depthFalloffStrength;
 
-        uint3 PADDING;
+        uint m_padding;
```

Comment:
Curious, what's the impact of this change given that the original struct was 4 byte padded already?

Reply:
Structs should be 16 byte aligned. This should be uint3.

Reply:
Not much impact as far as I can observed. Without the change, error message such as below would be given. In both cases the sample will still display without any observable graphical error:

```
06-09 06:45:33.120 22969 23047 I LMBR : $4[Error] (vkDebugMessage) - ERROR: [Validation] Code 0 : [
UNASSIGNED-CoreValidation-Shader-InconsistentSpirv ] Object: VK_NULL_HANDLE (Type = 0) | SPIR-V module not valid:
Structure id 10 decorated as Block for variable in Uniform storage class must follow standard uniform buffer layout
rules: member 1 at offset 4 is not aligned to 8
```

Reply:
I dont  understand this error. uint3 is also aligned to 8.

Reply:
reverted this after updating android version